backend/app/controllers/api/v1/invitations_controller.rb

```diff
@@ -0,0 +1,43 @@
+module API
+  module V1
+    class InvitationsController < BaseController
+      before_action :authenticate_user!, except: :update
+      before_action :fetch_user_by_email, only: :create
+      before_action :fetch_user_by_token, only: :update
+      before_action :require_light_account!
+
+      def create
+        authorize! :invite, @user
+
+        @user.invite! current_user
+        head :ok
+      end
+
+      def update
+        @user.with_lock do
+          @user.assign_attributes params.permit(:password, :first_name, :last_name, :ui_language)
+          @user.assign_attributes @user.invited_by&.attributes&.slice("role", "account_id") if @user.invited_by.present?
+          @user.save!
+          @user.accept_invitation!
```

Comment:
Meaning the user can log in already? Or maybe we should auto-login that user. I'm not sure. In the flow we have another step to send confirmation email, but first we disabled email confirmations for now, and second not sure if that is needed. I thought it would be more secure, for example if the attacker somehow obtains the invitation token, it still won't be possible to log in into that new account without having access to user's email account.

Reply:
That is something to discuss with frontend.

Reply:
Agree, I would prefer no to login user (will double check this and turn it off if it is on right now), because there is gonna be 2FA at future